### DISCUSSION TITLE
Add a new job, The Preacher.

### DISCUSSION THREAD

**Discussion by MaxIsJoe** [unitystation/unitystation#6983] [Ideas]
### Description

The preacher is a civilian/service job that is revolved around hallucinations and insanity. It's meant to be a job that adds a challenge for players and make maintenance a more risky area for antags to hide in.

- Players cannot communicate normally when outside of the chapel, they're not near the chaplain or if they're near a light source. They'll usually speak nonsense while shouting their god's name, yelling "HERASY" every now and then.
- - The preacher can choose to replace which religion and god they believe in with what the chaplain believes when joining the station.
- Preachers who pray in the chapel will have 10% chance get a random prediction after their 3rd prayer.
- - There will be a 50% chance they'll predict a minor event (like a gift getting sent from centcom), 35% for predicting major events (Asteroids colliding with the station) and 15% to predict who's an antagonist with an objective that intends to harm a crewmate or the station. (Custom objectives will be predicted regardless if they're harmful or not)
- - They will have a 100% chance to predict the existence of a cult as long as they have drew their first blood rune.
- - - Antags will not be named during predictions and the predictions will be vague, only showing hints on what the antag's job could be and their intentions (objectives) will also be vague. This is meant to make preachers keep an eye on multiple people or areas and to ICly cause some chaos by accusing an entire department of something negative.
- - - Preachers should NEVER have the ability to metagame, their entire existence is to be a silly job that has some "HA, told you so" moments with a few helpful quirks that could make the round more fun or less miserable in the late game for everyone.
- Preachers heal themselves by drinking holy water and getting hit by a holy book.
- Preachers cannot be converted into a cultist and will get gibbed when done so. They can however be converted to a revolutionary.
- Preachers can hunt for artifacts that only they can see in maintenance areas.
- - Each artifact collected gives preachers points to spend to gain a blessing
- - - blessings are spells/events that help (or harm) the station or the preacher himself.
- - - The more useful the blessing is the more expensive it will be
- - - Artifacts will be limited and can be disappear if a cultist found them, meaning that preachers will want to be more active in maintenance areas to look for them as they can offer blessings that can save the station in the late game or bring absolute chaos if the preacher is an antag.
- Preachers are insane, so they will see hallucinations if they don't pray or drink holy water.
- - The duration of how long they can go without praying or holy water is determined by how long they've spent time in the chapel + a variable number for each prayer or drink. 
- Gods can answer prayers by giving cookies to people who pray, Cookies earned by preachers have unique random effects.
- - These effects can range from curing the preacher's insanity to turning unprocessed/uncooked food into cooked dishes with a single touch. 

One more thing, prayers should work just like the original SS13. Meaning that admins will see all prayers players do, not just the preacher.
The preacher is special in this regard because they're completely devoted to the god they believe in ICly, so if the prayer from the preacher is good enough this will incentive admins to answer the preacher's prayers and make the round more fun (or chaotic) based on how creative the prayers are.
I also realize that many of the functionality needed for this job is not added yet but for now he can be a great silly job for RPing.

**Comment by wraith-54321** [comment]
giving them the ability to just say who an antag is is a REALLY bad idea, that would be abused 100% of the time

**Comment by wraith-54321** [comment]
its a fun idea but it sounds more like a midround then a job due to all the magic effects

**Comment by MaxIsJoe** [comment]
> giving them the ability to just say who an antag is is a REALLY bad idea, that would be abused 100% of the time

considering this is a 15% chance, I don't think so.
plus we can always vaguely tell preachers a detail about the antag, so for example; their god will tell them that "some of the scientists have a deadly sinful hate looming around them." which will lead to funny RP scenarios where preachers will accuse the science department of something and thins can escalate from there (for better or worse).

> its a fun idea but it sounds more like a midround then a job due to all the magic effects

Most of the magic effects is only available in the mid/late game because the preacher has to hunt for artifacts and wait for a cultist to draw their first blood rune to work. I don't think any person who late joins will be able to experience the full potential of the preacher if they late join especially on cult rounds.

**Comment by wraith-54321** [comment]
yes, that would be better, but just saying who an antag is would not not end well

**Comment by MaxIsJoe** [comment]
> yes, that would be better, but just saying who an antag is would not not end well

let me update the text then to make this clear.

**Comment by PerfectTangent** [comment]
This seems more like a chaplain discipline thing like /tg/ used to have than its own job. Also, discovering who antags are is kind of the psychiatrist's gimmick. Overall, some neat ideas, though

**Comment by PerfectTangent** [comment]
A mid round job like wraith said is also not a bad idea for this.
